Source/Core/Common/OpenXR.h

```diff
@@ -0,0 +1,120 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <openxr/openxr.h>
+
+#include <atomic>
+#include <memory>
+#include <string>
+#include <thread>
+#include <vector>
+
+#include "Common/MathUtil.h"
+#include "Common/Matrix.h"
+
+namespace OpenXR
+{
+bool IsInit();
+
+bool Init();
+bool Shutdown();
+
+void EventThreadFunc();
+
+class Session
+{
+  friend void EventThreadFunc();
+
+public:
+  Session() = default;
```

Comment:
This should be defaulted within the cpp file, given the class contains a few non-trivial objects.

Reply:
:+1: